index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    
+    <link href="styles.css" rel="stylesheet" type="text/css" />
+    <title>My Portfolio</title>
+
+    <script
+      src="https://kit.fontawesome.com/210152c720.js"
+      crossorigin="anonymous"
+    ></script>
+</head>
+<body id="hideOverFlow">
+    <header class="headers">
+        <a id="toblur1" class="mylogo dv-logo toblur">My Logo</a>
+        <nav id="menu_bar" class="nav-menu">
+            <i class="fas fa-bars"></i>
+        </nav>
+    </header>
+
+    <main>
+        <section class="about-me">
+            <div class="img-container">
+                
+            </div>
+            <div class="about-container">    
+                <div class="name">
+                    <h1>
+                        I'm Abdullateef<br />
+                        Glad To see<br />
+                        you!
+                    </h1>
+                </div>
+                <div class="name">
+                    <p class="abt-me">
+                        I’m a software developer! I can help you build a product , 
+                        feature or website Look through some of my work and experience! 
+                        If you like what you see and have a project you need coded, 
+                        don’t hestiate to contact me.
+                    </p>
+                </div>
+                <div class="name letsconnect">
+                    <p class="lets-connect">
+                        LET’S CONNECT
+                    </p>
+                </div>
+                <div class="name social-media-icons">
+                    <div class="twitter">
+                        <a href="#"><i class="fa fa-twitter"></i></a>
+                    </div>
+                    <div class="linkedin">
+                        <a href="#"><i class="fab fa-linkedin-in"></i></a>
+                    </div>
+                    <div class="medium">
+                        <a href="#"><i class="fab fa-medium"></i></a>
+                    </div>
+                    <div class="github">
+                        <a href="#"><i class="fa fa-github"></i></a>
+                    </div>
+                    <div class="angellist">
+                        <a href="#"><i class="fa fa-angellist"></i></a>
+                    </div>
+                </div>
```

Comment:
- [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)